lib/services/elasticsearch.js

```diff
@@ -880,36 +877,55 @@ class ElasticSearch extends Service {
    *
    * @param {String} index - Index name
    *
-   * @return {Promise.<Object>} { collections }
+   * @returns {Promise.<Array>} Collection names
    */
   listCollections (index) {
     return this.client.cat.indices({ format: 'json' })
       .then(({ body }) => {
-        const
-          esIndexes = body.map(({ index: esIndex }) => esIndex),
-          collections = this._extractCollections(esIndexes, index);
+        const esIndexes = body.map(({ index: esIndex }) => esIndex);
 
-        return {
-          collections
-        };
+        return this._extractCollections(esIndexes, index)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)